requirements-minimal.txt

```diff
@@ -1,5 +1,6 @@
+cryptography<=3.3.2
 emoji
 ocflib
 pyOpenSSL
 slackclient<2
-Twisted
+Twisted<=20.3.0
```

Comment:
Why did you need to restrict Twisted?

Could you add a comment for that and for the cryptography restriction (which I assume is related to the same `rustc` stuff mentioned in ocf/ircbot#193)

Reply:
Ah yeah I should leave comments. Twisted added incomplete types which messes with mypy, so I limited to an earlier release.